src/store/index.js

```diff
@@ -5,6 +5,12 @@ import { log } from './middleware';
 const reducer = (state = {}) => state;
 
 const initStore = () =>
-    createStore(reducer, compose(applyMiddleware(log), exposeSlices));
+    createStore(
```

Comment:
I had to `rm` the `package-lock.json` due to an `npm` bug with existing lockfiles and hoisting of shared deps. This pulled in a newer version of prettier, which added a heuristic for functional/compositional style chaining like this (request from RxJS team). Nothing in this code changes besides formatting.